pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -215,6 +216,12 @@ func (pas *PodAutoscalerStatus) MarkScaleTargetInitialized() {
 	podCondSet.Manage(pas).MarkTrue(PodAutoscalerConditionScaleTargetInitialized)
 }
 
+// ScaleTargetNotScaled returns true if the PodAutoscaler's scale target has failed
+// to scaled.
+func (pas *PodAutoscalerStatus) ScaleTargetNotScaled() bool {
+	return pas.GetCondition(PodAutoscalerConditionScaleTargetScaled).IsFalse()
+}
```

Comment:
`bool` result isn't accurate cause if the condition is unknown we still aren't considered 'scaled'